app/error_handlers.py

```diff
@@ -7,3 +7,10 @@
 def forbidden(e):
     app.logger.error(f"Forbidden Access{e}, route: {request.url}")  # pass in the error itself and the error
     return render_template("public/error_handlers/403.html")
+
+
+# Not Found Error
+@app.errorhandler(404)
+def forbidden(e):
```

Comment:
Should this function be renamed, e.g. as `not_found()`? At best, it's confusing to have two functions named `forbidden` in the same file.

Reply:
I  just got caught copying and pasting. 😅

Reply:
I know! 😉